components/table/SubTableRow.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { BuildingBlockType } from '../../service/types';
+import BBImage from './BuildingBlocksImage';
+
+type Props = {
+  buildingBlock: BuildingBlockType;
+};
+
+const SubTableRow = ({ buildingBlock }: Props) => {
+  return (
+    <div className='sub-table-row'>
+      <div></div>
+      <div className='sub-table-row-bb' data-testid='bb-name'>
+        <BBImage imagePath={buildingBlock.buildingBlock} />
+        <p>{(buildingBlock.buildingBlock).replace(/bb-|-/g, ' ')}</p>
```

Comment:
Instead of manipulating a string, I would prefer a service/function that handles mapping from identifiers received from the server to a user-friendly display string.

We could use the value from the server as a translation key (`bb-digital-registries`), if no translation is found we could fall back to string manipulation as you did in the current solution.

Reply:
I really like your idea! I'm going to handle it while working on a [ticket](https://govstack-global.atlassian.net/browse/TECH-507)